README.md

```diff
@@ -1,7 +1,7 @@
 # Calendar
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_calendar/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_calendar/report/html_report/)
+- [DEMO LINK](https://NazarShpylovyi.github.io/layout_calendar/)
+- [TEST REPORT LINK](https://NazarShpylovyi.github.io/layout_calendar/report/html_report/)
```

Comment:
Add these links to the PR description
<img width="920" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/64750958/93410a85-bb38-4e88-8288-0653785d4d01">